### PR TITLE
sam: add page

### PR DESCRIPTION
**Pull request by tmoon8730**
Added a page for the AWS SAM CLI tool

### CODE REVIEW

File: pages/linux/sam.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+> AWS Serverless Application Model (SAM) CLI.
+
+- Initialize a serverless application:
+
+`sam init`
+
+- Initialize a serverless application with a specific runtime:
+
+`sam init --runtime python3.7`
```
Comment (mebeim) [inline pages/linux/sam.md]:
```suggestion
`sam init --runtime {{python3.7}}`
```

Parameters entered by the user should be enclosed in double curly brackets.

Comment (tmoon8730) [reply pages/linux/sam.md]: Updated :+1:

File: pages/linux/sam.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Initialize a serverless application:
+
+`sam init`
+
+- Initialize a serverless application with a specific runtime:
+
+`sam init --runtime {{python3.7}}`
+
+- Package a SAM application (Alias of *aws cloudformation package*):
```
Comment (sbrl) [inline pages/linux/sam.md]: I'm not sure we need to mention an alias here. Perhaps we could write a page for `aws cloudformation` and put

```
> See also `aws cloudformation`.
```

as a 2nd description line?

File: pages/linux/sam.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# sam
+
+> AWS Serverless Application Model (SAM) CLI.
+
```
Comment (sbrl) [inline pages/linux/sam.md]: Perhaps we could add a new line here:

```
> Homepage: <https://github.com/awslabs/aws-sam-cli>
```

This is the new homepage syntax we defined in #2649.

File: pages/linux/sam.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- Build your Lambda function code:
+
+`sam build`
+
+- Run your serverless application locally:
+
+`sam local start-api`
+
+- Deploy an AWS SAM application (Alias of *aws cloudformation deploy*):
```
Comment (sbrl) [inline pages/linux/sam.md]: Same here.

File: pages/linux/sam.md (hunk 3 above)
Comment (tmoon8730) [reply pages/linux/sam.md]: Added. Thanks for the suggestion!

File: pages/linux/sam.md (hunk 2 above)
Comment (tmoon8730) [reply pages/linux/sam.md]: Great suggestion! For not I have removed the references all together as adding pages for the AWS CLI will take some time as it is pretty large. If there is interest in adding those pages I would be happy to do so and update this page to reference it.

File: pages/linux/sam.md (hunk 4 above)
Comment (tmoon8730) [reply pages/linux/sam.md]: See comment above

**Pull request merged**